executor/builder.go

```diff
@@ -320,12 +320,61 @@ func (b *executorBuilder) buildCheckIndex(v *plannercore.CheckIndex) Executor {
 	return e
 }
 
+func buildIndexLookUpChecker(b *executorBuilder, readerPlan *plannercore.PhysicalIndexLookUpReader,
+	readerExec *IndexLookUpExecutor) {
+	readerExec.keepOrder = true
```

Comment:
Why need to keep order?

Reply:
To save the handle offsets. But now I replace it with `isCheckOp` .